public/scss/partials/breach-detail.scss

```diff
@@ -447,7 +447,7 @@ button.fade-out {
   border-top: none;
   border-left: none;
   border-right: none;
-  border-bottom: 1px solid rgb(2, 80, 187, 0);
+  border-bottom: 1px solid rgba(2, 80, 187, 0);
```

Comment:
Good catch!
Wonder if we need some other grepping for any .css/.scss files that have `rgb(` strings and then count the parameters to see if it should be `rgb()` or `rgba()`.

Reply:
Stylelint caught this (and the others I found too)